modules/page-memory/src/main/java/org/apache/ignite/internal/pagememory/persistence/compaction/Compactor.java

```diff
@@ -216,6 +220,11 @@ void doCompaction() {
                 break;
             }
 
+            // TODO Expand the comment. https://issues.apache.org/jira/browse/IGNITE-23056
+            LOG.info("Starting new compaction round [files={}]", queue.size());
```

Comment:
I would suggest to use `isInfoEnabled` since the parameters used here are not calculated for free